.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+Thank you for your contribution!
+
+Please provide a summary of the changes and the motivation behind the change.
+
+## Reviewer Checklist
+
+- [ ] Documentation
+- [ ] Unit Tests
+- [ ] E2E Tests
```

Comment:
I'm not sure I'd include E2E tests in this list. I think we should _heavily_ push contributors/reviewers towards unit tests.

Obviously we'll need e2e's, but I don't think it makes sense on a PR-to-PR basis.

Reply:
Thoughts on changing this to `- [ ] E2E Tests (if appropriate)`

Reply:
WDYT about condensing down to this?

```
- [ ] Tests: unit tests (and e2es, if necessary)
```

In most cases I think we'll want unit tests only. So what's the best way to get that point across, while also reminding contributors and reviewers?

This:
- [x] Unit tests
- [ ] E2e tests (if appropriate)

Or this:
- [x] Tests: unit tests (and e2es, if appropriate)

The former might give the impression that a "good" PR checks all the boxes.
The latter might give the impression that a "good" PR just has to add some tests, doesn't matter which.

In fact a good PR will use _just_ a unit test if that's all that's required to test the change. And we'll only need an e2e to test the main happy paths or other functionality that just can't technically be covered in a unit test.

I think I lean toward the latter option, personally. Once we have a `CONTRIBUTING.md` we could have a section about testing, and we could then update the PR template to link the word `appropriate` to the CONTRIBUTING.md section that talks about what's appropriate for e2es.

As a follow-up, should we add a `CONTRIBUTING.md` that describes all this stuff in more detail?